build/stage-commit-validation.yaml

```diff
@@ -0,0 +1,33 @@
+# This stage is responsible for running the template to validate the commits of the PR
```

Comment:
What's this?

Reply:
This the stage file for the commit validation